app/controllers/textings_controller.rb

```diff
@@ -0,0 +1,108 @@
+class TextingsController < ApplicationController
+
+  def create
+    # Make a request to Textellent API. If request successful we update our DB
+    response = post_textellent(get_db_data)
+
+    if response['status'] == 'success'
+      recipient = TextingRecipient.find_by(phone_number: texting_params[:phone_number])
```

Comment:
So even if we do find a TextingRecipient with a matching phone number in the database, if for some reason the person's name changes, we may want to update the model in our database. This could happen in the very unlikely case where someone changes their phone number, and someone new gets that number, and both the old and the new owners of the number try to use this, but it could also happen if someone changes their name (e.g. they get married).

I know that this isn't the most likely thing to happen, but if it's only just a bit extra work, I think it'd be good to do it to future-proof the app.